tests/app/dao/test_templates_dao.py

```diff
@@ -95,35 +90,6 @@ def test_update_template(sample_service, sample_user):
     assert dao_get_all_templates_for_service(sample_service.id)[0].name == 'new name'
 
 
-def test_update_template_in_a_folder_to_archived(sample_service, sample_user):
```

Comment:
Hmm... should we maybe move this test instead of deleting it, or is this tested already in template's `test_rest.py`?

Reply:
good catch - I meant to check for that test. I've added one in template's test_rest.

Reply:
Thank you! 💖